`TestAccBackupGlobalSettings_basic` failing with `AccessDeniedException` in `us-west-2`

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

#### Test output:
```
------- Stdout: -------
=== RUN   TestAccBackupGlobalSettings_basic
=== PAUSE TestAccBackupGlobalSettings_basic
=== CONT  TestAccBackupGlobalSettings_basic
    global_settings_test.go:18: Step 1/4 error: Error running apply: exit status 1
        
        Error: error setting Backup Global Settings (*******): AccessDeniedException: 
        	status code: 403, request id: 2a65bcbf-47d2-4bba-8a71-8e251f798e6c
        
          on terraform_plugin_test.tf line 2, in resource "aws_backup_global_settings" "test":
           2: resource "aws_backup_global_settings" "test" {
        
        
--- FAIL: TestAccBackupGlobalSettings_basic (45.45s)
FAIL
```

--- a/internal/service/backup/global_settings_test.go
+++ b/internal/service/backup/global_settings_test.go
@@ -18,6 +18,7 @@ func TestAccBackupGlobalSettings_basic(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck: func() {
 			acctest.PreCheck(t)
+			acctest.PreCheckOrganizationManagementAccount(t)
 			testAccPreCheck(t)
 		},
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),

